A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2949,65 +2957,79 @@ class FMCMainDisplay extends BaseAirliners {
     }
 
     trySetFlapsTHS(s) {
-        if (s) {
-            let validEntry = false;
-            let nextFlaps = this.flaps;
-            let nextThs = this.ths;
-            let [flaps, ths] = s.split("/");
-
-            // Parse flaps
-            if (flaps && flaps.length > 0) {
-                if (!/^\d+$/.test(flaps)) {
-                    this.addNewMessage(NXSystemMessages.formatError);
-                    return false;
-                }
+        if (s === FMCMainDisplay.clrValue) {
+            this.flaps = NaN;
+            this.ths = NaN;
+            SimVar.SetSimVarValue("L:A32NX_TO_CONFIG_FLAPS", "number", 0);
+            SimVar.SetSimVarValue("L:A32NX_TO_CONFIG_FLAPS_ENTERED", "bool", false);
+            SimVar.SetSimVarValue("L:A32NX_TO_CONFIG_THS", "degree", 0);
+            SimVar.SetSimVarValue("L:A32NX_TO_CONFIG_FLAPS_ENTERED", "bool", false);
```

Comment:
This line ended up twice (see 2 lines up). Not sure if it’s from master or merge conflict fix-up?

Reply:
Looks like it was like this before.

![image](https://user-images.githubusercontent.com/1857322/106503496-0085c100-64c6-11eb-8214-4250e06684a8.png)